style.mss

```diff
@@ -2,26 +2,335 @@ Map {
-   Fallback is a regular style. */
-@bold-fonts:    "DejaVu Sans Bold", "Arundina Bold", "Arundina Sans Bold", "Padauk Bold", "TSCu_Paranar Bold",
-                "DejaVu Sans Book", "Arundina Regular", "Arundina Sans Regular", "Padauk Regular", "Khmer OS Metal Chrieng Regular",
-                "Mukti Narrow Regular", "gargi Medium", "TSCu_Paranar Regular", "Tibetan Machine Uni Regular",
-                "Droid Sans Fallback Regular", "Unifont Medium", "unifont Medium";
-
-/* An oblique or italic style is available for various (mostly latin-like) scripts. Oblique and italic style are not heavier than
-   regular style. But they are distinguishable from regular style. Fallback is a regular style. Because many (mostly asian) scripts
-   do not have an oblique or italic style by nature, be aware that for a considerable number of labels this style will make no
-   difference to the regular style. */
-@oblique-fonts: "DejaVu Sans Oblique", "Arundina Italic", "Arundina Sans Italic", "TSCu_Paranar Italic",
-                "DejaVu Sans Book", "Arundina Regular", "Arundina Sans Regular", "Padauk Regular", "Khmer OS Metal Chrieng Regular",
-                "Mukti Narrow Regular", "Gargi Regular", "TSCu_Paranar Regular", "Tibetan Machine Uni Regular",
-                "Droid Sans Fallback Regular", "Unifont Medium", "unifont Medium";
+
+/*
+About fonts:
+
+Noto is a font family that wants to cover most of Unicode with a harmonic
+design across various scripts. We use Noto for most text, with some support
+for backward-compatibility and Unifont as fallback.
+
+By order:
+
+1. Noto Sans is available for most scripts and it is used here. The list in is
+alphabetical order. Note that the “Noto Sans” base font MUST be before the
+“Noto Sans CJK *” fonts, because the CJK fonts contain non-Noto latin characters
+that should not be used. Noto Serif and Noto Mono are available for various
+(mostly latin-like) scripts. They are ignored here in favour of the respective
+Noto Sans variant. For fonts that are known to have an UI variant (a variant
+designed for user interface with tighter vertical metrics) the list contains
+both - the normal and the UI variant - but prefers the UI variant.
+
+2. Noto provides three variants of Arabic: Noto Kufi Arabic, Noto Naskh Arabic
+and Noto Nastaliq Urdu. Noto Kufi Arabic is the one that is most similiar to a
+Sans style, so this one is used while both other variants are ignored. Arabic
+fonts are placed behind Sans fonts because they might re-define some commonly
+used signs like parenthesis or quotation marks, and the arabic design should
+not overwrite the standard design.
+
+3. Noto provides two variants of Emoji: Noto Color Emoji and Noto Emoji. The
+colour variant is a SVG flavoured OpenType font that contains coloured emojis.
+This is not useful in cartography, so we use the “normal” monochromatic
+Noto Emoji.
+
+4. The list still includes the previously used font selection of this style
+to remain backward-compatible.
```

Comment:
If we're happy with Noto we can cut out the previous list. I might still keep DejaVu around for compatibility, but not the others.